docs/release-instruction.md

```diff
@@ -0,0 +1,32 @@
+# Release instructions
+
+## Pre-release checks
+
+These should run successfully:
+* unit-tests,
+* `go mog tidy` (should not change any files);
+* `./prepare.sh /path/to/neofs-api/on/your/machine` (should not change any files).
+
```

Comment:
```
* gofmt -w ./
* golangci-lint run ./...
```

Reply:
> is it a good idea to write such instructions without mentioning the version of such utils? it also relates to ./prepare.sh and protoc

This instruction is targeted for repository maintainers, I think they can figure out the version of standard go tools we use. golangci-lint configured with fixed amount of linters as well. As for `prepare.sh`, any protoc version update trigger file changes, so it is not a problem too.

Reply:
ok